### PR TITLE
[METAED-1505] - Adds support for DS 5.0 final

### PR DESCRIPTION
**Pull request by DavidJGapCR**
Based on conversation with Steven Arnold on Slack:
`7.1 is the final ODS API release. 7.1 will support both DS 4.0 and 5.0.`

### CODE REVIEW

**Comment by bradbanister** [comment]
@DavidJGapCR Did you confirm ODS/API 7.0 is also compatible with DS 5.0.0?

**Comment by DavidJGapCR** [comment]
> @DavidJGapCR Did you confirm ODS/API 7.0 is also compatible with DS 5.0.0?

Yes. I talked to @stevenarnold-ed-fi. He said: "7.1 will support both DS 4.0 and 5.0."

**Comment by DavidJGapCR** [comment]
> > @DavidJGapCR Did you confirm ODS/API 7.0 is also compatible with DS 5.0.0?
> 
> Yes. I talked to @stevenarnold-ed-fi. He said: "7.1 will support both DS 4.0 and 5.0."

Ah wait... You said 7.0. I'll ask.

**Comment by DavidJGapCR** [comment]
Okay, I asked Steven Arnold. The answer is yes.

**Comment by stevenarnold-ed-fi** [comment]
> Okay, I asked Steven Arnold. The answer is yes.

@DavidJGapCR @bradbanister I think I misread the slack message. ODS/API 7.0 will support DS 5.0-pre.2 (and DS 4.0), 7.1 will be the only version that supports DS 5.0

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"displayName": "MetaEd IDE",
 	"description": "Ed-Fi MetaEd IDE for Visual Studio Code",
 	"license": "SEE LICENSE IN LICENSE.md",
-	"version": "4.2.1",
+	"version": "4.2.2-dev.22",
```
Comment (stephenfuqua) [inline package.json]: The latest one in Azure Artifacts is `dev.21`.

Comment (bradbanister) [reply package.json]: I pushed before I was done committing 🙄

**Review by bradbanister — APPROVED** [review]

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**